Tools/scripts/generate_manifest.py

```diff
@@ -12,6 +12,39 @@
 FIRMWARE_TYPES = ["AntennaTracker", "Copter", "Plane", "Rover", "Sub"]
 RELEASE_TYPES = ["beta", "latest", "stable", "stable-*"]
 
+# mapping for board names to brand name and manufacturer
+brand_map = {
+    'Pixhawk4' : ('Pixhawk 4', 'Holybro'),
```

Comment:
We may want to abstract this information into a separate `.py` so we can use it in both waf and here (and possibly other places).  This would mean this `brand_map` doesn't get out of date.

Reply:
agreed